app/services/payout/potential_payment/uphold_service.rb

```diff
@@ -54,29 +55,31 @@ def perform
       fee_probi = wallet.channel_balances[channel.details.channel_identifier].fees_probi # fee = balance - probi
       total_probi += probi
 
-      unless should_only_notify?
-        PotentialPayment.create(
-          payout_report_id: @payout_report.id,
-          name: "#{channel.publication_title}",
-          amount: "#{probi}",
-          fees: "#{fee_probi}",
-          publisher_id: @publisher.id,
-          channel_id: channel.id,
-          kind: PotentialPayment::CONTRIBUTION,
-          address: "#{uphold_connection.address}",
-          url: "#{channel.details.url}",
-          uphold_status: uphold_connection.status,
-          reauthorization_needed: uphold_connection.uphold_access_parameters.blank?,
-          uphold_member: uphold_connection.is_member?,
-          uphold_id: uphold_connection.uphold_id,
-          wallet_provider_id: uphold_connection.uphold_id,
-          wallet_provider: PotentialPayment.wallet_providers['uphold'],
-          suspended: @publisher.suspended?,
-          status: @publisher.last_status_update&.status,
-          channel_stats: channel.details.stats,
-          channel_type: channel.details_type
-        )
-      end
+      potential_payments << PotentialPayment.new(
+        payout_report_id: @payout_report&.id,
+        name: "#{channel.publication_title}",
+        amount: "#{probi}",
+        fees: "#{fee_probi}",
+        publisher_id: @publisher.id,
+        channel_id: channel.id,
+        kind: PotentialPayment::CONTRIBUTION,
+        address: "#{uphold_connection.address}",
+        url: "#{channel.details.url}",
+        uphold_status: uphold_connection.status,
+        reauthorization_needed: uphold_connection.uphold_access_parameters.blank?,
+        uphold_member: uphold_connection.is_member?,
+        uphold_id: uphold_connection.uphold_id,
+        wallet_provider_id: uphold_connection.uphold_id,
+        wallet_provider: PotentialPayment.wallet_providers['uphold'],
+        suspended: @publisher.suspended?,
+        status: @publisher.last_status_update&.status,
+        channel_stats: channel.details.stats,
+        channel_type: channel.details_type
+      )
+    end
+
+    unless should_only_notify?
+      potential_payments.each(&:save)
```

Comment:
Review with eV: Write out a payout message and update some metabase dashboards to display this result.

Reply:
https://github.com/brave-intl/publishers/pull/2733/commits/2ed9d5a73a67dba6a923d9521d98e045d2af9a6a